guides/micronaut-data-jdbc-repository/java/src/main/java/example/micronaut/ApplicationConfiguration.java

```diff
@@ -0,0 +1,8 @@
+package example.micronaut;
+
+import javax.validation.constraints.NotNull;
+
+public interface ApplicationConfiguration {
+
+    @NotNull Integer getMax();
+}
```

Comment:
Why the need for an interface and an implementation? Isn't it simpler to just write:

```java
@ConfigurationProperties("application")
public interface ApplicationConfiguration {   
    @NotNull Integer getMax();
}
```

Reply:
I prefer pure interfaces decoupled from `@ConfigurationProperties` implementations.

Reply:
Just seems to add more verbosity to the example IMO